src/widget/wbasewidget.cpp

```diff
@@ -139,63 +139,72 @@ void WBaseWidget::setControlParameterRightUp(double v) {
 }
 
 void WBaseWidget::updateTooltip() {
-    // If we are in developer mode, update the tooltip.
+    // Remove leading and trailing whitespace/line breaks from the tooltip.
+    QString base = baseTooltip().trimmed();
+
+    // If we are in developer mode, expand the tooltip.
     if (CmdlineArgs::Instance().getDeveloper()) {
         QStringList debug;
         fillDebugTooltip(&debug);
 
-        QString base = baseTooltip();
         if (!base.isEmpty()) {
-            debug.append(QString("Tooltip: \"%1\"").arg(base));
+            debug.append(QStringLiteral("Tooltip: \"%1\"").arg(base));
         }
-        m_pWidget->setToolTip(debug.join("\n"));
+        m_pWidget->setToolTip(debug.join(QChar('\n')));
+    } else {
+        m_pWidget->setToolTip(base);
```

Comment:
This means QWidget::setTooltip() is now called on every tooltip event which is redundant.
Please move it back into the CmdLine..dev parantheses